test/e2e/storage/persistent_volumes.go

```diff
@@ -319,6 +325,260 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 		})
 	})
 
+	ginkgo.Describe("CSI Conformance", func() {
+
+		var pvols e2epv.PVMap
+		var claims e2epv.PVCMap
+
+		ginkgo.AfterEach(func(ctx context.Context) {
+			framework.Logf("AfterEach: deleting %v PVCs and %v PVs...", len(claims), len(pvols))
+			errs := e2epv.PVPVCMapCleanup(ctx, c, ns, pvols, claims)
+			if len(errs) > 0 {
+				errmsg := []string{}
+				for _, e := range errs {
+					errmsg = append(errmsg, e.Error())
+				}
+				framework.Failf("AfterEach: Failed to delete 1 or more PVs/PVCs. Errors: %v", strings.Join(errmsg, "; "))
+			}
+		})
+
+		ginkgo.It("should run through the lifecycle of a PV and a PVC", func(ctx context.Context) {
```

Comment:
ConformanceIt is wrapper function for ginkgo It.  Adds "[Conformance]" tag and makes static analysis easier.

```suggestion
		ginkgo.It("should run through the lifecycle of a PV and a PVC", func(ctx context.Context) {
		ginkgo.ConformanceIt("should run through the lifecycle of a PV and a PVC", func(ctx context.Context) {
```

Reply:
The above change is planned as per the [Conformance test requirements](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-requirements) but it needs to do the following first

> it is stable and runs consistently (e.g., no flakes), and has been running for at least two weeks